data/virtualization/add_pci_bridge.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# add_pci_bridge.sh 
+# Add pci devices to guest .xml to enable hotplugging. This script can be expanded to add custom number
+# of devices depending on specific tests cases.
+# Maintainer: Teodor Baev <teodor.baev@suse.com>
+
+machine="$1"
+xml="$2"
+
+if [[ $machine == "q35" ]]; then
+	# add 10 pcie-root-port devices for hotplugging
+	for n in {1..10}
```

Comment:
I am curious about the minimum required pcie-root-port for our hotplugging test and how you decide to add 10 ports?